### PR TITLE
solution

### PR DESCRIPTION
**Pull request by NikitaNikolaitsev**

### CODE REVIEW

File: app/test_split_integer.py [hunk 1]
```diff
@@ -2,20 +2,22 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(8, 2)) == 8
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert split_integer(8, 2) == [4, 4]
```
Comment (DevSpaciX) [inline app/test_split_integer.py:9]: Add an error message. It's good when the user understands what the problem is

File: app/test_split_integer.py [hunk 2]
```diff
@@ -2,20 +2,22 @@
...
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    assert split_integer(17, 4) == [4, 4, 4, 5]
 
 
 def test_should_add_zeros_when_value_is_less_than_number_of_parts() -> None:
-    pass
+    assert split_integer(1, 2) == [0, 1]
+# Do I specifically need to break one of the tests?
+# If I break it, then pytest app/ won't pass
```
Comment (DevSpaciX) [inline app/test_split_integer.py:23]: The test failed because of pass param . Try to add error messages as I say below , and create another PR

**Review by DevSpaciX — CHANGES_REQUESTED** [review]

File: app/test_split_integer.py [hunk 3]
```diff
@@ -2,20 +2,25 @@
...
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    assert split_integer(17, 4) == [4, 4, 4, 5], \
+        "Parts should be sorted when they are not equal"
 
 
 def test_should_add_zeros_when_value_is_less_than_number_of_parts() -> None:
-    pass
+    assert split_integer(1, 2) == [0, 1], \
```
Comment (DevSpaciX) [inline app/test_split_integer.py:25]: Remove this breaking lines ( \ ) . Try to use more beautiful variants

**Review by DevSpaciX — CHANGES_REQUESTED** [review]

File: app/test_split_integer.py (hunk 3 above)
Comment (NikitaNikolaitsev) [reply app/test_split_integer.py:25]: ok only this problem?

**Review by DevSpaciX — APPROVED** [review]
Great !

**Review by DevSpaciX — CHANGES_REQUESTED** [review]
Make all test passed

**Pull request closed without merging**